Dockerfile

```diff
@@ -2,41 +2,53 @@ FROM centos:6
 
 MAINTAINER IoT team
 
-RUN yum update -y && yum install -y wget
-RUN wget http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
-RUN yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm
-RUN yum install -y rpm-build git java-1.7.0-openjdk java-1.7.0-openjdk-devel
-ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+ENV DB_HOST localhost
+ENV KEYPASS_VERSION 1.0.0
 
-# Install Maven
-RUN wget -c http://ftp.cixug.es/apache/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.zip
-RUN unzip -oq apache-maven-3.2.5-bin.zip
-RUN cp -rf apache-maven-3.2.5 /opt/maven
-RUN ln -fs /opt/maven/bin/mvn /usr/bin/mvn
 
-## Install MySQL (client at least?)
-RUN yum -y install mysql
+RUN \
+    # Install dependencies
+    yum update -y && yum install -y wget  && \
+    wget http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm && \
+    yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm  && \
+    yum install -y rpm-build git java-1.7.0-openjdk java-1.7.0-openjdk-devel
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.7.0-openjdk.x86_64
```

Comment:
You may move this ENV sentence above the previous RUN or below the next, so you can use a single RUN sentence for both (neither of them use Java, so it doesn't seem important to define it in the middle).